<launch>
	<!--SIMULATION-->
	<arg name="fcu_url" default="udp://:14540@localhost:14557"/> <!-- Change for appropriate Simulation host IP -->
	<arg name="gcs_url" default=""/>
	<arg name="tgt_system" default="1" />
	<arg name="tgt_component" default="1" />
	<arg name="log_output" default="screen" />

	<include file="$(find mavros)/launch/node.launch">
		<arg name="pluginlists_yaml" value="$(find mavros)/launch/px4_pluginlists.yaml" />
		<arg name="config_yaml" value="$(find mavros)/launch/px4_config.yaml" />

		<arg name="fcu_url" value="$(arg fcu_url)" />
		<arg name="gcs_url" value="$(arg gcs_url)" />
		<arg name="tgt_system" value="$(arg tgt_system)" />
		<arg name="tgt_component" value="$(arg tgt_component)" />
		<arg name="log_output" value="$(arg log_output)" />
	</include>

	<include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
		<arg name="port" value="9090"/>
	</include>
	<node name="unitycontrol" pkg="px4_control" type="drone_unity.py" output="screen">
		<param name="delays" value="false" />
		<param name="delays_dir" value="./delays" />
		<param name="delays_thresh" value="100" />
	</node>
</launch>
